tools/aws_lab_setup/roles/control_node/templates/ansible.cfg.j2

```diff
@@ -1,5 +1,12 @@
 [defaults]
 connection = smart
 timeout = 60
+{% if networking is defined %}
```

Comment:
What is thinking behind this? It's not apparent to me why this is necessary.

Reply:
- networking stores their inventory here: /home/{{ username }}/networking-workshop/lab_inventory/hosts

- the essentials mode stores their hosts here: {{ control_node_inventory_path }} which corresponds to-> ~{{ username }}/lightbulb/lessons/lab_inventory/{{ username }}-instances.txt

ideally I would like to change the essentials mode to match the networking one, which is easier to write directions and lab guides to since they will be the same for every student.  This allows the student to cut and paste directions and not get stuck (since their username is different).  This was something that evolved based on survey feedback from teaching the classes.  I didn't change the essentials mode b/c I have not taught that class, and I figured it was outside of my expertise and responsibility.

Reply:
I don't see why the generated static inventory can't be in one common generic location like `~/hosts`. See #107. 

Please bear in mind that the provisioner we "borrowed" was developed under a very different scope and circumstances and is equally as flawed to the extent.

Reply:
OK, so what can we do here that is expedient and helps to solve this issue? Because if there's a fairly straightforward change we can make here, it's a good time for it -- but I don't think it's fair to ask for a large / time-consuming change to fix an issue that is pre-existing. *Especially* if this is a stopgap anyway and we're planning on factoring all of this stuff out into roles.

Reply:
I guess what you're recommending is factoring the hardcoded IDs out into the docs, and just saying "replace the vpc-xxxxx values with your own"?

Reply:
@tima I am glad you agree, I think we should just add an issue, assign to me and I can work with Gerald to change essentials to just use the hosts naming inventory nomenclature.  The only reason I won't switch it right now with this PR is I would have to go change all the instructions that Roland just did a PR with.  I am pretty sure the lab guides point to the old one... that way it would simplify both workshops and the .ansiblecfg  template